stanovy_vpsFree.cz.md

```diff
@@ -305,107 +368,104 @@ systému Spolku nejméně 10 pracovních dnů před začátkem platnosti.
 1. Hospodář je člen Rady odpovědný za:
   1. správu finančního a hmotného majetku,
   2. řádné vedení účetnictví,
-  3. účetní operace podle platných právních předpisů.
+  3. účetní operace podle platných právních předpisů
 2. Hospodáře volí Rada ze svých členů.
 
 ### §20 Předseda
 
-1. Jménem Spolku podepisuje Předseda; v jeho nepřítomnosti člen pověřený Radou.
-Podepisování za Spolku se uskutečňuje tak, že k názvu Spolku se připojí jméno
-a podpis předsedy. V případě, že podepisuje pověřený člen Rady, je povinen
-doložit pověření od Rady.
-2. Předseda je odpovědný za celkovou koordinaci, vedení a provozní činnost
```

Comment:
Je dobry napad tohle odstranit? Takhle bylo jasne, kdo ma podpisove pravo...

Reply:
Čekal bych že to bude spíš součástí zápisu v rejstříku, tak jako je to u firem, ale nenašel jsem to tam. Kde jinde by to mohlo být uvedeno?

Reply:
@snajpa dává ti to vysvětlení smysl?

Reply:
Dava, ale takhle explicitne se to hodi, napr. kdyz se jedna o nejaky pujcky/bankovni sluzby, tem jde o co nejvetsi jistotu a tohle neskodi :)

Reply:
pro tyhle případy je obvykle rozhodující to, co je v rejstříku a v rejstříku to bude napsaný zhruba ve smyslu "předseda jedná jménem rady samostatně". 

Stanovy nikoho takhle moc zajímat nebudou (vlastní zkušenost).

Ale secondary layer "jistoty" imho asi snad ničemu neuškodí, zkusim to tam nějak naformulovat zpátky. Nicméně si  furt myslim, že předseda nemůže z principu jednat jménem spolku, protože pak by to byl de-facto druhej statutár vedle Rady.

Reply:
resolved by 4924e22, @snajpa vyhovuje takhle?

Reply:
idealka